src/sentry/tempest/models.py

```diff
@@ -0,0 +1,35 @@
+from django.conf import settings
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import DefaultFieldsModel, FlexibleForeignKey, region_silo_model
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+
+
+class MessageType(models.TextChoices):
+    ERROR = "error"
+    WARNING = "warning"
+    SUCCESS = "success"
+    INFO = "info"
+
+
+@region_silo_model
+class TempestCredentials(DefaultFieldsModel):
+    # Contains sensitive information which we don't want to export/import - it should be configured again manually
+    __relocation_scope__ = RelocationScope.Excluded
+
+    created_by_id = HybridCloudForeignKey(settings.AUTH_USER_MODEL, null=True, on_delete="SET_NULL")
+    project = FlexibleForeignKey("sentry.Project", on_delete=models.CASCADE)
```

Comment:
I just want to confirm whether this should be unique and if there are any other uniques we might consider? Generally I think it's worth thinking through any uniqueness constraints ahead of time, because it's a pain to fix them retroactively

Reply:
No, we expect to have multiple client ids / secrets per project. I should have put it into description of the PR.

Only usage of this table would be to retrieve all the id/secret pairs for a project (for which we already have an index as a foreign key), and to add new things. Not even update of data would be allowed, only deletion.

P.S. sorry, i didn't know that you get notifications on every push, i was going to respond to your comment above and re-request a review, but wanted to make sure that CI is passing first 😬

Reply:
Np, just wanted to make sure it wasn't missed, thanks!